### PR TITLE
Issue #196 expose server.port

### LINKED ISSUE #196: expose server.port
## Summary

What: Expose `server.port`

Why: Self explanatory

## Acceptance Criteria

- [x] `server.port` is accessible

## Example Pseudo Code (for implementation)

```typescript
// server.ts

  /**
   * @description
   *     The port of the Deno server.
   *
   * @property number port
   */
  public port: number = 1447;
  ...
  ...
  ...
  public async run(options: HTTPOptions): Promise<void> {
    if (!options.hostname) {
      options.hostname = this.hostname;
    }
    if (!options.port) {
      options.port = this.port;
    }
  ...
  ...
  ...
  public async runTLS(options: HTTPSOptions): Promise<void> {
    if (!options.hostname) {
      options.hostname = this.hostname;
    }
    if (!options.port) {
      options.port = this.port;
    }
  ...
  ...
  ...
```

### PR DESCRIPTION
**Pull request by Guergeiro**
This closes #196

### CODE REVIEW

File: src/http/server.ts [hunk 1]
```diff
@@ -473,7 +487,7 @@ export class Server {
       options.hostname = this.hostname;
     }
     if (!options.port) {
-      options.port = 1447;
+      options.port = this.port;
     }
     this.hostname = options.hostname;
```
Comment (crookse) [inline src/http/server.ts:492]: need `this.port = options.port` here

File: src/http/server.ts [hunk 2]
```diff
@@ -505,7 +519,7 @@ export class Server {
...
```
Comment (crookse) [inline src/http/server.ts:524]: need `this.port = options.port` here

File: src/http/server.ts (hunk 1 above)
Comment (Guergeiro) [reply src/http/server.ts:492]: [Done](https://github.com/drashland/deno-drash/pull/198/commits/be96d393acc92dad7cda2f7489606a40ac92b6c4).

File: src/http/server.ts (hunk 2 above)
Comment (Guergeiro) [reply src/http/server.ts:525]: [Done](https://github.com/drashland/deno-drash/pull/198/commits/be96d393acc92dad7cda2f7489606a40ac92b6c4).

**Pull request merged**